src/pages/edge/adobe-journey-optimizer-decisioning/assurance-setup.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Setting up Decisioning plugins in Assurance
+description: This document shows user how to setup Assurance for Mobile SDK.
+keywords:
+- Assurance
+- Decisioning
+---
+
+# Setting up Decisioning plugins in Assurance
+
+This document shows user how to setup Assurance for Mobile SDK.
+
+## Setup
+
+1. To get started you will need Adobe Experience Platform Assurance Extension to be enabled and set up in the application. Follow the instruction provided in [**Assurance Overview**](../../home/base/assurance/index.md) for setup.
```

Comment:
you will need Adobe Experience Platform Assurance Extension to be enabled and set up in the application -> you will need Adobe Experience Platform Assurance extension to be installed in your mobile application.